lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -303,14 +342,17 @@
                                 languages: self.languages,
                                 teamMembershipDetailUrl: self.teamMembershipDetailUrl
                             });
-                            var teamJoinView = new TeamJoinView({
+
+                            var TeamProfileActionsView = new TeamProfileHeaderActionsView({
```

Comment:
We use `TeamProfileActionsView ` convention for views, models and collections names.